pkg/regressionallowances/regressions_4.17.json

```diff
@@ -0,0 +1,52 @@
+[
+  {
+    "JiraComponent": "Management Console",
+    "TestID": "openshift-tests:f9552d2b4b817d19bdcd666a7b8d1ece",
+    "TestName": "[bz-Management Console] clusteroperator/console should not change condition/Available",
+    "JiraBug": "https://issues.redhat.com/browse/OCPBUGS-39580",
+    "ReasonToAllowInsteadOfFix": "Test flaked in 4.16, became more strict in 4.17. Condition/Available goes from true to false exactly once for an undetermined DNS issue, possibly attributable to misconfigured retry in console operator.  Seen only on Vsphere OVN UPI, not see on IPI.",
```

Comment:
> became more strict in 4.17

This is the most important part of the justification. Can you provide a link to the commit that changed this for future us?